### PR TITLE
Bootstrapping the project

### PR DESCRIPTION
**Pull request by Senjuti256**
Added the bats submodules required and created the basic project infrastructure.

### CODE REVIEW

**Comment by Senjuti256** [comment]
@otaviof @vdemeester @Aneesh-M-Bhat PTAL and LMKF.

**Review by Aneesh-M-Bhat — APPROVED** [review]
LGTM, you can start with the oc implementation as I see it.

File: .gitmodules
Comment (otaviof) [inline .gitmodules:1]: Could you please make sure the `.gitmodules` and the modules themselves are on a single commit?

**Review by otaviof — CHANGES_REQUESTED** [review]
It looks good! We only need to split the git-submodules into a standalone commit, please.

File: .gitmodules
Comment (Senjuti256) [reply .gitmodules:1]: As far as I remembered .gitmodules and the modules themselves belonged to a single commit itself. The rest of the files were added through two separate commits. Later before creating this PR I squashed all these 3 commits together.

Comment (Senjuti256) [reply .gitmodules:1]: Should I delete rest of the files from this commit except the submodules and .gitmodules @otaviof ?

Comment (otaviof) [reply .gitmodules:1]: I mean, you should have two commits, one with the project submodules, and another one with the rest of the files.

**Comment by Senjuti256** [comment]
I have done the required changes @otaviof . PTAL and LMK your feedback.

File: values.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+images:
+  bash: registry.access.redhat.com/ubi8-minimal:latest
+  openshift-cli: registry.redhat.io/openshift-pipelines/pipelines-cli-tkn-rhel8
```
Comment (vdemeester) [inline values.yaml]: This is the `tkn` image (and probably not the public address), not the `oc` image.

**Review by vdemeester — COMMENTED** [review]
We need a small CI running on this.

File: values.yaml (hunk 1 above)
Comment (Senjuti256) [reply values.yaml]: I have updated the image for openshift-client.

**Comment by Senjuti256** [comment]
> We need a small CI running on this.

How to do this @vdemeester ? Any reference please.

**Review by vdemeester — APPROVED** [review]

**Comment by vdemeester** [comment]
Let's get this merge, check the CI and iterate from it.

**Pull request merged**